examples/data_readers.ipynb

```diff
@@ -615,6 +612,57 @@
     "data = dp.Data(filepath, options=options)\n",
     "data.data.head()"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b50679ea",
+   "metadata": {},
+   "source": [
+    "## Intro to `GraphData` data reader\n",
+    "\n",
+    "This tutorial will focus on how to use the data reader class: `GraphData`. The `GraphData` class is used for reading the delimited data from a CSV file into a `NetworkX` Graph object. This is all in an effort to prepare the data automaticaly for `GraphProfiler` class to then profile graph data. \n",
+    "\n",
+    "The DataProiler keys off of common graph naming conventions in the column header row. E.G. from `dataprofiler/tests/csv/graph_data_csv_identify.csv`\n",
+    "```\n",
+    "node_id_dst, node_id_src, continuous_weight, categorical_status\n",
+    "108,289,7.4448069,9\n",
+    "81,180,3.65064207,0\n",
+    "458,83,5.9959787,10\n",
+    "55,116,4.63359209,79\n",
+    "454,177,5.76715529,11\n",
+    "429,225,4.79556889,3\n",
+    "```\n",
+    "\n",
+    "Options for the `GraphData` are exactly the same as `CSVData`.\n",
+    "\n",
+    "\n",
+    "Example implementation of `GraphData`:\n",
+    "```python\n",
+    "import dataprofiler as dp\n",
+    "from dataprofiler.data_readers.graph_data import GraphData\n",
+    "\n",
+    "data = dp.Data(graph_file)\n",
+    "data = GraphData(graph_file)\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "838db976",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from dataprofiler.data_readers.graph_data import GraphData\n",
+    "\n",
+    "# use data reader to read delimited data \n",
+    "data_folder = \"../dataprofiler/tests/data\"\n",
+    "graph_file = \"csv/graph_data_csv_identify.csv\"\n",
+    "\n",
+    "data = GraphData(os.path.join(data_folder, graph_file))\n",
+    "print(data.data.edges)\n",
```

Comment:
`data.data` isn't really an option for networkX object .... I'm not seeing an attirbute returning the data as a table. So I think either `data.data.edges` or `nodes` are the option for demonstrating the graph data